libcloud/loadbalancer/drivers/gce.py

```diff
@@ -338,9 +338,12 @@ def _node_to_member(self, node, balancer):
         # would be found if it was there.
         if hasattr(node, 'name'):
             member_id = node.name
-            member_ip = node.public_ips[0]
         else:
             member_id = node
+
```

Comment:
@allardhoeve Thx a lot for the feedback. 

1/ node object : In fact the node object is good.  The node IP has indeed an empty "publicIP" which lead to the error since we try to access the first element. Maybe I did not fully understand your feedback ?
2/ test case : I can definitively try to add a test case (will be a good experience for me). 

Will update the PR with a test case ASAP and wait for your clarification about the point 1/

Thx again guys for the feedback.